src/hooks/useUser.tsx

```diff
@@ -41,6 +42,14 @@ const useUser = () => {
     }
   }, [isError]);
 
+  useEffect(() => {
```

Comment:
I believe this will be called an unnecessary amount of times, just if you reload the page or whatever. Could you move this to only on login?

Reply:
Thanks, removed.